app/src/main/java/de/westnordost/streetcomplete/quests/AbstractOsmQuestForm.kt

```diff
@@ -47,7 +47,6 @@ import de.westnordost.streetcomplete.view.confirmIsSurvey
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
-import kotlinx.serialization.encodeToString
```

Comment:
Why is this import removed when nothing about `encodeToString` in this file changed? Does it still work correctly?

Reply:
it was an unused import